core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -488,6 +488,17 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val STORAGE_REMOTE_SHUFFLE_MAX_DISK =
+    ConfigBuilder("spark.storage.remote.shuffle.maxDisk")
+      .doc("Maximum disk space to use to store shuffle blocks before rejecting remote " +
+        "shuffle blocks. Rejecting remote shuffle blocks means that an exec will not receive any " +
+        " shuffle migrations, and if there are no execs avaialble for migration then " +
+        s"decommissioning will block unless ${STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH.key} " +
```

Comment:
Will block? I think decommission will stop soon rather than block in that case.

Reply:
I actually don't understand why the sentence "if there are no execs available ...." appended here. I think it's not necessarily needed here.